ui/app/Makefile

```diff
@@ -23,19 +23,22 @@ format: elm-env $(ELM_FILES)
 	@$(DOCKER_CMD) elm-format --yes $(ELM_FILES)
 
 test: elm-env
+	@$(DOCKER_CMD) rm -rf elm-stuff/generated-code
 	@$(DOCKER_CMD) elm-format $(ELM_FILES) --validate
 	@$(DOCKER_CMD) elm-test
 
 dev-server:
-	elm-reactor
+	elm reactor
 
 script.js: elm-env format $(ELM_FILES)
 	@echo ">> building script.js"
-	@$(DOCKER_CMD) elm make src/Main.elm --yes --output $(TEMPFILE_JS)
-	@$(DOCKER_CMD) uglifyjs $(TEMPFILE_JS) --compress unused --mangle --output $(@)
+	@$(DOCKER_CMD) rm -rf elm-stuff
```

Comment:
This forces us to re-download all the dependencies on each build, right? Is this necessary?

Reply:
I know, I had some weird errors with the messed up `elm-stuff`, perhaps because `elm-test` does some weird things with `test-dependencies`,  it's still pretty fast though.

Reply:
Ok, sounds good to me. We can still change it, in case it is worth optimizing.